site/assets/styles/styles.css

```diff
@@ -448,7 +477,7 @@ a {
 }
 
 .blocks.blocks--even {
-    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
solves some overlap problems? - can you bring that up tomorrow at our meeting?

Reply:
just ask I might forget